opentelemetry-appender-tracing/src/lib.rs

```diff
@@ -1 +1,149 @@
+//! # OpenTelemetry-Appender-Tracing
+//!
+//! This crate provides a bridge between the [`tracing`](https://docs.rs/tracing/latest/tracing/) crate and OpenTelemetry logs.
+//! It converts `tracing` events into OpenTelemetry `LogRecords`, allowing applications using `tracing` to seamlessly integrate
+//! with OpenTelemetry logging backends.
+//!
+//! ## Background
+//!
+//! Unlike traces and metrics, OpenTelemetry does not provide a dedicated logging API for end-users. Instead, it recommends using
+//! existing logging libraries and bridging them to OpenTelemetry logs. This crate serves as such a bridge for `tracing` users.
+//!
+//! ## Features
+//!
+//! - Converts `tracing` events into OpenTelemetry [`LogRecords`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#log-and-event-record-definition)
+//! - Integrates as a [`Layer`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/trait.Layer.html)
+//!   from [`tracing-subscriber`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/), allowing
+//!   to be used alongside other `tracing` layers, such as `fmt`
+//! - Automatically attaches OpenTelemetry trace context (`TraceId`, `SpanId`, `TraceFlags`) to logs
```

Comment:
This is behind feature flag. Probably good to mention that.

Reply:
This is Otel Trace context, which is *not* under feature flag. (I guess you meant using the tracing's own context, which is under feature flag. all ff are covered at the end of the doc.)